monero-outreach-docs/translations/it/quickfacts/QuickFacts_it.md

```diff
@@ -0,0 +1,127 @@
+# MONERO
+
+Monero è una valuta digitale sicura, privata e non tracciabile
+disponibile per tutti.
+
+# IN BREVE
+
+Gli utilizzatori possono spendere in 
+maniera sicura, nessuno può vedere il 
+loro saldo o tenere traccia della loro attività.
+
+Revisione del 14/11/2018 - [Versione stampabile](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)
+Creato per la comunità da Monero Outreach
+
+[getmonero.org](https://getmonero.org/it/)
+
+
+## Punti chiave
+
+#### DECENTRALIZZAZIONE
+
+In quanto _[progetto open-source](https://github.com/monero-project/monero)_ gestito e finanziato da un _[team decentralizzato](https://www.getmonero.org/it/community/team/)_ di sviluppatori e membri della comunità, esso non può essere censurato. La maggior parte dei collaboratori è formata da volontari e la comunità è distribuita in tutto il globo.
+
+#### SICUREZZA
+
+In quanto criptovaluta decentralizzata, la sicurezza di Monero viene garantita da una ampia rete di utilizzatori in ogni parte del mondo. Le transazioni vengono confermate attraverso un meccanismo di consenso distribuito e quindi registrate in modo immutabile sulla _[blockchain](https://it.wikipedia.org/wiki/Blockchain)_.
+
+#### SCALABILITÀ
+
+La _[grandezza dinamica del blocco](https://it.wikipedia.org/wiki/Blockchain#Struttura_del_blocco)_ cambia in base al volume delle transazioni per garantire commissioni più basse e transazioni più veloci. Un volume di transazioni più alto porta ad un limite maggiore sulla grandezza del blocco mentre un volume più basso porta ad un limite minore sulla grandezza del blocco. Le _[Bulletproofs](https://web.stanford.edu/~buenz/pubs/bulletproofs.pdf)_ aiutano inoltre la scalabilità riducendo significativamente la dimensione della transazione e le commissioni, facendo così risaltare Monero nel panorama delle criptovalute.
+
+#### RESISTENZA ALLA CENSURA
+
+Le transazioni sulla blockchain Monero non rivelano un utilizzatore in particolare o una identità nel mondo reale. Pertanto gli utilizzatori non possono essere soggetti a censura o a _[controlli di capitale](https://argomenti.ilsole24ore.com/parolechiave/controlli-capitali.html)_.
+
+#### PRIVACY
+
+Monero usa schemi crittografici sofisticati come le _[firme ad anello](https://getmonero.org/it/resources/moneropedia/ringsignatures.html)_, le _[transazioni confidenziali ad anello](https://www.mycryptopedia.com/monero-ring-confidential-transactions-ringct/)_ e gli _[indirizzi nascosti](https://www.mycryptopedia.com/everything-need-know-stealth-addresses/)_ per offuscare l'origine, l'ammontare e la destinazione di ogni transazione.
```

Comment:
Missing two links:

- _[transazioni confidenziali ad anello](https://www.mycryptopedia.com/monero-ring-confidential-transactions-ringct/)_
Correct link:
https://www.getmonero.org/it/resources/moneropedia/ringCT.html

- _[indirizzi nascosti](https://www.mycryptopedia.com/everything-need-know-stealth-addresses/)_
Correct link:
https://www.getmonero.org/it/resources/moneropedia/stealthaddress.html